doc/getstarted/basic_usage/index_en.rst

```diff
@@ -70,20 +68,19 @@ Now that everything is ready, you can train the network with a simple command li
         paddle train --config=trainer_config.py --save_dir=./output --num_passes=30
  
 
-This means that PaddlePaddle will train this network on the synthectic dataset for 30 passes, and save all the models under path ``./output``. You will see from the messages printed out during training phase that the model cost is decreasing as time goes by, which indicates we are getting a closer guess.
+This means that PaddlePaddle will train this network on the synthetic dataset for 30 passes, and save all the models under the path :code:`./output`. You will see from the messages printed out during training phase that the model cost is decreasing as time goes by, which indicates we are getting a closer guess.
 
 
 Evaluate the Model
 -------------------
 
-Usually, a different dataset that left out during training phase should be used to evalute the models. However, we are lucky enough to know the real answer: ``w=2, b=0.3``, thus a better option is to check out model parameters directly.
+Usually, a different dataset that left out during training phase should be used to evaluate the models. However, we are lucky enough to know the real answer: :math:`w=2, b=0.3`, thus a better option is to check out model parameters directly.
```

Comment:
Usually, test dataset (different from training dataset) is needed to evaluate the models. However, we have already known the exact function  :math: `w=2, b=0.3`, thus we can check out the model parameters directly.